dnf/cli/commands/history.py

```diff
@@ -176,33 +173,25 @@ def _hcmd_redo(self, extcmds):
 
     def _history_get_transactions(self, extcmds):
         if not extcmds:
-            logger.critical(_('No transaction ID given'))
-            return None
+            raise dnf.cli.CliError(_('No transaction ID given'))
 
         old = self.base.history.old(extcmds)
         if not old:
-            logger.critical(_('Not found given transaction ID'))
-            return None
+            raise dnf.cli.CliError(_('Not found given transaction ID'))
```

Comment:
Please fix the message, you can actually include the id in the message. See this: https://github.com/rpm-software-management/ci-dnf-stack/pull/916#discussion_r545139121, I'd prefer to keep my superior message :slightly_smiling_face:

Reply:
Fixed